### PR TITLE
bugfix: fix did-resize modifier conditional

### LINKED ISSUE #97: Cannot set properties of undefined (setting 'font')
when _shouldUseNativeLineClampCSS or _shouldUseNativeTextOverflowCSS is true then this.canvasContext is not created causing an error in _calculateTargetWidth

### PR DESCRIPTION
**Pull request by thegilby**
## Summary
Fixes #97.

#92 added the ability to install the `did-resize` modifier conditionally. This PR fixes an upstream error caused by this change when `_shouldUseNativeTextOverflowCSS` is `true` by adding an additional conditional to the `didResize()` getter.

## Miscellaneous
- Add test case for Native Text Overflow and did resize
- Update `htmlSafe` import in test file to `@ember/template`
- Fix `test-conatiner` typos in test file.
- Add `@babel/plugin-proposal-private-property-in-object` and `@ember/string` as devDeps to fix issues when integrating with applications and a couple ember-try scenarios.

## Testing Done
- Repo tests pass 
- Application tests pass when integrated via `yarn link`

### CODE REVIEW

File: addon/components/line-clamp.js [hunk 1]
```diff
@@ -225,7 +225,10 @@ export default class LineClampComponent extends Component {
    * @return {DidResizeModifier | null}
    */
   get didResize() {
-    return this._shouldUseNativeLineClampCSS ? null : DidResizeModifier;
+    return this._shouldUseNativeLineClampCSS ||
+      this._shouldUseNativeTextOverflowCSS
+      ? null
+      : DidResizeModifier;
```
Comment (drewlee) [inline addon/components/line-clamp.js:231]: Minor (optional) suggestion for legibility, otherwise LGTM.
```suggestion
    const shouldUseNativeCSS =  this._shouldUseNativeLineClampCSS || this._shouldUseNativeTextOverflowCSS;
    return shouldUseNativeCSS ? null : DidResizeModifier;
```

Comment (thegilby) [reply addon/components/line-clamp.js]: @drewlee Good call. Updated!

**Comment by thegilby** [comment]
> @lstrrs Would appreciate it if you could review and get this published as a new release. Thanks!

@lstrrs Bumping this. Would appreciate a review and a new release after this lands. Thanks!

**Review by lstrrs — APPROVED** [review]
